src/scales/schemes.js

```diff
@@ -144,7 +144,7 @@ const ordinalSchemes = new Map([
 function scheme9(scheme, interpolate) {
   return ({length: n}) => {
     if (n === 1) return [scheme[3][1]]; // favor midpoint
-    if (n === 2) return [scheme[3][0], scheme[3][2]]; // favor extrema
+    if (n === 2) return [scheme[5][1], scheme[5][3]]; // favor extrema, avoid lightness
```

Comment:
```suggestion
    if (n === 2) return [scheme[3][1], scheme[3][2]]; // favor darker
```

What about this instead?

Reply:
I like it! Updated [here](https://observablehq.com/@observablehq/boolean-colors-pr). 

This will now fail the test that `colorSchemesOrdinal must match snapshot`. I'm not sure how to go about updating a test that matches HTML output files (do I change those directly)?

Reply:
You can regenerate the snapshots by saying `rm -rf test/output` and then `yarn test`. Or you can copy over the `-changed` file to replace the snapshot.

Reply:
(This is covered in [CONTRIBUTING.md](https://github.com/observablehq/plot/blob/main/CONTRIBUTING.md).)

Reply:
Ah, great - thank you. Will do that now.